addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -443,7 +444,7 @@ export async function activateCropper(image, aspectRatio, dataset) {
 export async function loadImageInfo(img, rpc, attachmentSrc = '') {
     const src = attachmentSrc || img.getAttribute('src');
     // If there is a marked originalSrc, the data is already loaded.
-    if (img.dataset.originalSrc || !src) {
+    if ((img.dataset.originalSrc && img.dataset.mimetypeBeforeConversion) || !src) {
```

Comment:
@loco-odoo Don't remember: is not that the thing we fully reverted in stable? Should not it be added back in stable (in its fixed version)?

Reply:
@qsm-odoo, This fix should be added back in stable but it can not be done now. Indeed, [this commit](https://github.com/odoo/odoo/commit/3528f2c3f9d3b30266f3421672986d202c724a76) fixed the problem of the absolute URL being stored after an image move, however, clients that did the move before the fix still have an absolute URL stored in their db. If I am not mistaken, @sobo-odoo is working on a task to convert the absolute URL to relative ones and it is after the merge of its task that this fix can be merged. Indeed, the goal of this fix is to perform the `loadImageInfo` operation for the images that do not have the `mimetypeBeforeConversion` attribute. To do so, the `src` argument has to be correct in order for the `/web_editor/get_image_info` rpc call to correctly return `original`.

Reply:
@loco-odoo We won't convert user URLs back to relative ones but make it so our code is more robust to work with absolute ones. In any case, this fix should be made in stable then forward-ported. If it has to wait for something before being merged, so be it.

@sobo-odoo Are you working on something like that? (make it so the code does work with absolute URLs). If not, decide which one of you is going to fix this :+1: Arguably, this could be made in the same PR, maybe even same commit as this stable fix about `mimetypeBeforeConversion`.

Reply:
@qsm-odoo @loco-odoo I didn't start yet but I was planning to do it (it was lower in my priority list) :slightly_smiling_face: If it is urgent, I will try to do it this week :)

Reply:
@loco-odoo it's done :wink: See [this PR](https://github.com/odoo/odoo/pull/136951) (for the master version of the fix)